src/main/scala/diplomacy/Nodes.scala

```diff
@@ -560,7 +747,9 @@ class IdentityNode[D, U, EO, EI, B <: Data](imp: NodeImp[D, U, EO, EI, B])()(imp
   } 
 }
 
-// EphemeralNodes disappear from the final node graph
+/**
+ * EphemeralNodes disappear from the final node graph
```

Comment:
i never understood what this meant. So you put them beween two nodes and then it's like they were never there...?

Reply:
Note that I didn't touch this at all besides changing the plain comment to a Scaladoc comment. I have not yet gotten to a point where I understand what an EphemeralNode is or what it should be used for.